sources/web/datalab/polymer/components/auth-panel/auth-panel.ts

```diff
@@ -47,7 +47,10 @@ class AuthPanel extends Polymer.Element {
 
   ready() {
     super.ready();
-    GapiManager.loadGapi(this._signInChanged.bind(this));
+    GapiManager.listenForSignInChanges(this._signInChanged.bind(this))
+      .catch(() => {
+        // TODO: handle errors authenticating
```

Comment:
Do we want to call console.error in all these places while waiting to decide how to handle them?